libcloudforensics/providers/gcp/internal/compute.py

```diff
@@ -397,13 +394,12 @@ def ListDiskByLabels(
 
     disk_service_object = self.GceApi().disks()
     return self.__ListByLabel(  # type: ignore
-        labels_filter, disk_service_object, filter_union)
+        labels_filter, disk_service_object, filter_union)  # type: ignore
 
-  def __ListByLabel(
-      self,
-      labels_filter: Dict[str, str],
-      service_object: 'googleapiclient.discovery.Resource',
-      filter_union: bool) -> Dict[str, Union['GoogleComputeInstance', 'GoogleComputeDisk']]:  # pylint: disable=line-too-long
+  def __ListByLabel(self,
```

Comment:
Use single leading underscores if you want to declare private functions (also in `__SshConnection`)

Reply:
Done. Neither of these methods belong to this PR, if they were touched it is the yapf stuff, I changed them in all places, they can't be referenced from other places so should be fine